docker/conf/docker-compose.github-actions.review.yml.dist

```diff
@@ -0,0 +1,91 @@
+version: "3.4"
+
+services:
+    postgres:
+        restart: always
+        image: postgres:12.1-alpine
+        volumes:
+            - postgres-conf:/var/lib/postgresql/conf
+        environment:
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=shopsys
+        command:
+            - postgres
+            - -c
+            - config_file=/var/lib/postgresql/conf/postgres.conf
+        networks:
+            - default
+        labels:
+            - traefik.enable=false
+
+    webserver:
+        restart: always
+        image: nginx:1.13-alpine
```

Comment:
Can we use newer version?

Reply:
We can but it is version we use in all of our docker-compose files